brainstormforce/header-footer-elementor
# EPS-971: Load text domain during init action to prevent early translation loading error.

### Description

Getting a notice - 
The _load_textdomain_just_in_time function was called incorrectly. Translation loading for the header-footer domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the init action or later. (This message was added in version 6.7.0.)


### Screenshots

https://bsf.d.pr/v/XXvEap

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
1. Install Loco translate
2. Select the language further 
3. Set the language in your setting
4. Check for all the settings for the UAE, it should remain intact.
5. Screencast - https://bsf.d.pr/v/XXvEap

### Checklist:
